ui/app/styles/core/alert-banner.scss

```diff
@@ -0,0 +1,6 @@
+.alert-banner-message-body {
+  border: 0;
+  margin-top: $spacing-xxs;
```

Comment:
it looks like the `border` and `margin-top` here have the same values as the original `message-body` class. in that case, what do you think about extending `message-body` so these styles aren't duplicated? this way if we make any other styling changes to all messages (perhaps a new font or font weight, etc), they'll apply here too. i.e.:

```
.alert-banner-message-body {
@extend .message-body
color: $black;
}
```

Reply:
Ah, that's a great idea.  I didn't know you could do that.  Thanks!

Reply:
@noelledaley ... um. 🤔  this doesn't seem to work because `message-body` color is overriding the black here.  I would have to use `!important` after black to make it work, and that doesn't seem like an optimal solution.